backend/Dockerfile

```diff
@@ -8,10 +8,13 @@
 # BASE
 FROM php:7.4-fpm-buster as base
 
-WORKDIR /usr/app
+ARG UID=1000
+ARG GID=1000
+
+RUN usermod -u $UID www-data
+RUN groupmod -g $GID www-data
 
 COPY --from=mlocati/php-extension-installer /usr/bin/install-php-extensions /usr/bin/
-COPY --from=composer:2 /usr/bin/composer /usr/bin/composer
```

Comment:
? why

Reply:
I've changed the order of this command to line 39 in the same file.